bee-message/src/util/mod.rs

```diff
@@ -1,8 +1,17 @@
 // Copyright 2021 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
+mod map;
+
+#[cfg(not(feature = "std"))]
+pub(crate) use map::{Map, Set};
+#[cfg(feature = "std")]
+pub(crate) use std::collections::{HashMap as Map, HashSet as Set};
```

Comment:
Should this move to the bottom of the `use` list? Usually `std` imports would go there.